src/Sentry.Log4Net/SentryAppender.cs

```diff
@@ -65,41 +66,48 @@ protected override void Append(LoggingEvent loggingEvent)
                 }
             }
 
-            var exception = loggingEvent.ExceptionObject ?? loggingEvent.MessageObject as Exception;
-            var evt = new SentryEvent(exception)
+            if (loggingEvent.Level > Breadcrumb)
```

Comment:
Perhaps we should have 2 event levels here like we do on NLog, Serilog, MEL.
The decision wouldn't have to be: Either breadcrumb or event.
We actually store the errors as breadcrumbs too so they get sent on the next event as part of the breadcrumb.